tools/aliveness/src/main.rs

```diff
@@ -0,0 +1,358 @@
+use std::{
+    collections::HashMap,
+    env::args,
+    mem::size_of,
+    net::{Ipv4Addr, SocketAddr, SocketAddrV4},
+    ptr::read,
+    time::Duration,
+};
+
+use aliveness::{
+    service_manager::SystemServices, AlivenessState, BEACON_HEADER, BEACON_MULTICAST_GROUP,
+    BEACON_PORT,
+};
+use color_eyre::eyre::{bail, eyre, ContextCompat, Result, WrapErr};
+use configparser::ini::Ini;
+use futures_util::stream::StreamExt;
+use hula_types::{Battery, RobotState};
+use log::{error, info};
+use tokio::{
+    io::AsyncReadExt,
+    net::{UdpSocket, UnixStream},
+    select, spawn,
+    sync::{mpsc, oneshot},
+    task::JoinHandle,
+    time::interval,
+};
+use tokio_util::sync::CancellationToken;
+use zbus::{
+    zvariant::{OwnedObjectPath, Value},
+    Connection, MatchRule, MessageStream, MessageType, Proxy,
+};
+
+const HULA_SOCKET_PATH: &str = "/tmp/hula";
```

Comment:
This magic value is duplicated in HULA and Aliveness. In PR #47 (insert your Hitman reference here) we tried to deduplicate such magic values by moving them into a central shared crate. Do we want to deduplicate this one in addition? Since this PR may get merged sooner, feel free to cherry-pick Paul's changes and then he will rebase on your branch (or `main` if merged).

Reply:
Do you mean as an entry of `crates/hulks_os_version/src/lib.rs`? Then this crate should probably be renamed, to something like `constants`. On that note, should `SDK_VERSION` in `crates/repository` then also be moved there (or has it even the same value as the `HULKS_OS_VERSION` and can be merged with it)?

Reply:
I disagree to fast-tracking the changes and features of paul to this PR. If he is going to introduce such a crate with constants, he can also include this `/tmp/hula` constant. But this is a totally different feature set as julian is introducing with this PR

Reply:
IMHO someone has to deduplicate and it would be weird if Paul needs to fix an unrelated HULA constant in his PR. The PR that gets merged first should introduce this new crate to `main` and the second one will extend it by more values. I don't know which comes first. Probably Julian's.

Yes, it needs to be renamed. I like the `constants` suggestion. If you have suggestions in Paul's PR, feel free to write a review comment over there. 😉

Reply:
The `HULA_SOCKET_PATH` constant is now moved to the new crate `constants`.